### PR TITLE
Update public to private

### PR DESCRIPTION
**Pull request by 597365581**
Update public to private

**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [x] If there is a related issue, make sure it is linked to this PR.
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

### CODE REVIEW

**Comment by 597365581** [comment]
please help to review

**Comment by 597365581** [comment]
@tdas tdas

**Comment by dennyglee** [comment]
Could you clarify the reason behind switching this from `public` to `private`?  As well, as reminded to me by @ravivj-db 
```
This is autogenerated java client code using openapi-generator sbt plugin. There are no getters in the class for the variables for which you are trying to change the declaration from public to private. 
```

TL;DR this change may hamper any code trying to access these variables.

File: clients/java/src/main/java/io/unitycatalog/client/ServerVariable.java [hunk 1]
```diff
@@ -7,9 +7,9 @@
  */
 @javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaClientCodegen", comments = "Generator version: 7.5.0")
 public class ServerVariable {
```
Comment (tdas) [inline clients/java/src/main/java/io/unitycatalog/client/ServerVariable.java:9]: this is generated code. so we do not manually modify this code.

**Review by tdas — COMMENTED** [review]
We will not accept any changes to generated code. Please close your PR.

**Comment by tdas** [comment]
Closing this PR. If you have other changes you want to propose in non-generated code, please make another PR.
